newsletter-november-2019.rst

```diff
@@ -0,0 +1,136 @@
+.. post:: November 04, 2019
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – November 2019
+#########################################
+
+`It's...<https://www.youtube.com/watch?v=ydyXFUmv6S4>`__ not Monty Python's Flying Circus. Sorry. It is, however, the November edition of the newsletter!
```

Comment:
Is this too niche? I'm leaning towards taking it out.

Reply:
I'm gonna take this out.

Reply:
Yeah, I'd take it out.